### PR TITLE
Changed "client.xml" to "client.yml". Fixed for Symfony3

### PR DESCRIPTION
**Pull request by eduardoledo**
Changed client configuration in compliance with Symfony3 Service Loader

### CODE REVIEW

File: DependencyInjection/DoctrineCouchDBExtension.php [hunk 1]
```diff
@@ -89,16 +91,16 @@ private function clientLoad($config, ContainerBuilder $container)
...
                 $connection,
             ))
         ;
 
         if (isset($connection['logging']) && $connection['logging'] === true) {
             $def = new Definition('Doctrine\CouchDB\HTTP\Client');
-            $def->setFactoryService(sprintf('doctrine_couchdb.client.%s_connection', $name));
-            $def->setFactoryMethod('getHttpClient');
+            $def->setFactory(sprintf('doctrine_couchdb.client.%s_connection', $name), 'getHttpClient');
+//            $def->setFactoryMethod('getHttpClient');
```
Comment (alcaeus) [inline DependencyInjection/DoctrineCouchDBExtension.php]: If it's not necessary, remove it.

File: DependencyInjection/DoctrineCouchDBExtension.php [hunk 2]
```diff
@@ -63,8 +65,8 @@ public function getConfiguration(array $config, ContainerBuilder $container)
 
     private function clientLoad($config, ContainerBuilder $container)
     {
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('client.xml');
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('client.yml');
```
Comment (alcaeus) [inline DependencyInjection/DoctrineCouchDBExtension.php]: Please keep XML as format. Symfony has no requirement for yaml and this is an unnecessary change.

Comment (eduardoledo) [reply DependencyInjection/DoctrineCouchDBExtension.php]: In hindsight it might be an overkill. I'll make those changes and submit it again.

**Comment by beberlei** [comment]
Merged another symfony3 PR instead.
